.github/labeler.yml

```diff
@@ -0,0 +1,29 @@
+'is: documentation':
+- changed-files:
+  - any-glob-to-all-files: '{**/docs/**,**/README.md}'
+
+'affects: webhost':
+- changed-files:
+  - all-globs-to-any-file: 'WebHost.py'
+  - all-globs-to-any-file: 'WebHostLib/**/*'
+
+'affects: core':
+- changed-files:
+  - all-globs-to-any-file:
+    - '!*Client.py'
```

Comment:
CommonClient is considered core (at the moment at least), so we shouldn't block it here. `**/!(*CommonClient).py` would suffice, or putting it in the any-glob-to-any-file.

Reply:
Good catch, will add it to the any-to-any list. I was wondering the same about bizhawk client as well actually

Reply:
BizhawkClient and SNIClient would also fall under this I believe.

Reply:
Bizhawk and SNI are strange. They have their own maintainers, kind of, but the component is shared between multiple worlds. I don't think they are truly core in the same sense as other things (also bizhawk should probably be moved to worlds/_bizhawk and sni possibly to worlds/_snes or something). So I'm fine with them not being labeled as `core`.